packages/expo-secure-store/android/src/main/java/expo/modules/securestore/SecureStoreModule.java

```diff
@@ -110,12 +117,24 @@ private void setItemImpl(String key, String value, ReadableArguments options, Pr
       // back a value.
       if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
         KeyStore.SecretKeyEntry secretKeyEntry = getKeyEntry(KeyStore.SecretKeyEntry.class, mAESEncrypter, options);
-        encryptedItem = mAESEncrypter.createEncryptedItem(value, keyStore, secretKeyEntry);
-        encryptedItem.put(SCHEME_PROPERTY, AESEncrypter.NAME);
+        mAESEncrypter.createEncryptedItem(promise, value, keyStore, secretKeyEntry, options, mAuthenticationHelper.getDefaultCallback(), new PostEncryptionCallback() {
+          @Override
+          public void run(Promise promise, Object result) throws JSONException {
+            JSONObject obj = (JSONObject) result;
+            obj.put(SCHEME_PROPERTY, AESEncrypter.NAME);
+            saveEncryptedItem(promise, obj, prefs, key);
+          }
+        });
```

Comment:
It should be possible to be converted into lambda (the same for other `EncryptionCallback` and `PostEncryptionCallback` implementations):
```suggestion
        mAESEncrypter.createEncryptedItem(promise, value, keyStore, secretKeyEntry, options, mAuthenticationHelper.getDefaultCallback(), (callbackPromise, result) -> {
          JSONObject obj = (JSONObject) result;
          obj.put(SCHEME_PROPERTY, AESEncrypter.NAME);
          saveEncryptedItem(callbackPromise, obj, prefs, key);
        });
```

Reply:
Done in https://github.com/expo/expo/pull/14512/commits/531d1a47ddd80d2b5758771176f205a0c590ca9a.